packages/oui-datagrid/src/datagrid-column-builder.service.js

```diff
@@ -87,6 +87,17 @@ export default class DatagridColumnBuilder {
         return {};
     }
 
+    static hasAttribute (element, attributeName) {
+        const attributes = [];
+        Array.from(element.attributes).forEach(attribute => attributes.push(attribute.name));
+        Object.keys(element.dataset).forEach(dataKey => attributes.push(dataKey));
```

Comment:
Why do you store it ? You don't use it and I don't found it in the current version of the service.

Reply:
My bad, I missread the code. Ignore my comment.